### DISCUSSION TITLE
The application menu is missing

### DISCUSSION THREAD

**Discussion by sotchenkov** [elementary/triage#475] [Q&A] [answered]
Good afternoon! 👋🏻
I ran into a little problem - my application menu is missing:
![el](https://github.com/elementary/triage/assets/40219980/71ba32b3-f627-4c57-8051-0619f8f1450a)

And I'm trying to get it back using https://github.com/elementary/applications-menu but I can't find the libswitchboard-3 package:
```bash
apt search libswitchboard
Sorting... Done
Full Text Search... Done
libswitchboard-2.0-0/jammy,now 6.0.2~r1198+pkg78~ubuntu7.1 amd64 [installed,automatic]
  Modular Settings Hub - shared libraries

libswitchboard-2.0-dev/jammy,now 6.0.2~r1198+pkg78~ubuntu7.1 amd64 [installed]
  Modular Settings Hub - development files
```

Can you tell me what I'm doing wrong?

**Comment by lenemter** [comment, marked as answer]
I think you can install applications-menu using `sudo apt install slingshot-launcher`. Can you try it?

> I can't find the libswitchboard-3 package

libswitchboard-3 is in-development gtk4 version of our settings library and it's not available in stable OS 7.1

**Reply by sotchenkov** [reply]
Yes. It worked for me. Thank you very much :)
